src/whisper_at_train/run_combined_training.sh

```
@@ -7,11 +7,22 @@
 #SBATCH --output=./log/%j_as.txt
 
-set -x
-# comment this line if not running on sls cluster
-#. /data/sls/scratch/share-201907/slstoolchainrc
-source /home/taemyung_heo/.cache/pypoetry/virtualenvs/whisper-at-z6hdRBdT-py3.10/bin/activate
-export TORCH_HOME=../../pretrained_models
+set -euxo pipefail
+
+# Environment variables with defaults
+WHISPER_AT_VENV="${WHISPER_AT_VENV:-/home/taemyung_heo/.cache/pypoetry/virtualenvs/whisper-at-z6hdRBdT-py3.10/bin/activate}"
+WHISPER_AT_BASE="${WHISPER_AT_BASE:-$(dirname $(dirname $(pwd)))}"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)